spec/steps/medicaid/insurance_current_type_spec.rb

```diff
@@ -0,0 +1,101 @@
+require "rails_helper"
+
+RSpec.describe Medicaid::InsuranceCurrentType do
+  describe "Validations" do
+    context "each insured household member has an insurance type" do
+      it "is valid" do
+        member = create(:member,
+                        is_insured: true,
+                        insurance_type: "Medicaid")
+
+        member_without_insurance = create(:member,
+                                          is_insured: false,
+                                          insurance_type: nil)
+
+        step = Medicaid::InsuranceCurrentType.new(
+          members: [member, member_without_insurance],
+        )
+
+        expect(step).to be_valid
+      end
+    end
+
+    context "an insured household member is missing an insurance type" do
+      it "is invalid" do
+        member = create(:member,
+                        is_insured: true,
+                        insurance_type: "Medicaid")
+
+        insured_member_without_type = create(:member,
+                                             is_insured: true,
+                                             insurance_type: nil)
+
+        step = Medicaid::InsuranceCurrentType.new(
+          members: [member, insured_member_without_type],
+        )
+
+        expect(step).not_to be_valid
+      end
+    end
+  end
+
+  describe "#insured_members_needing_insurance" do
+    context "with insured members needing insurance" do
+      it "returns only insured members needing insurance" do
```

Comment:
this is good -- we should be doing this check on *all* steps, perhaps

Reply:
this is a good point